### ISSUE TITLE
Bug 54111816: [React Native Gallery -> Home]: The Name property is containing characters in the private Unicode range.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#518]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Fast Pass
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Home’ button present in left navigation pane and invoke it.
Now run fast pass and observe the issue.
Actual Result:
The Name property is containing characters in the private Unicode range.

Expected Result:
The Name property must not contain any characters in the private Unicode range.

User Impact:
Users with visual impairment who rely on screen reader will be misguided if narrator announces incorrect control type of the item.
 
Link to ADO bug: https://microsoft.visualstudio.com/OS/_workitems/edit/54111816

**Comment by Yajur-Grover** [comment]
This should be able to be resolved by changing the value of the `accessibilityLabel` property of the affected component - if not, please feel free to contact me for guidance.

**Comment by anupriya13** [comment]
https://github.com/microsoft/react-native-gallery/pull/545
